src/libraries/Common/tests/System/Net/Http/HttpClientHandlerTest.RemoteServer.cs

```diff
@@ -77,8 +77,7 @@ public async Task UseDefaultCredentials_SetToFalseAndServerNeedsAuth_StatusCodeU
             }
         }
 
-        [OuterLoop("Uses external servers")]
-        [ActiveIssue("https://github.com/dotnet/runtime/issues/37669", TestPlatforms.Browser)]
+        [OuterLoop("Uses external servers", TestPlatforms.Any & ~TestPlatforms.Browser)]
```

Comment:
For the browser test I pass `Content-MD5-Skip` and because I could handle it differently in xharness hosted server, this makes lot of tests pass. All those tests are testing other useful things besides integrity of the payload.